### DISCUSSION TITLE
React Native support

### DISCUSSION THREAD

**Discussion by META-DREAMER** [family/connectkit#68] [Ideas]
Adding support for React Native / Expo would make this the defacto wallet connection UX for universal dapps that share code between web and mobile (e.g. with https://solito.dev)

**Comment by lochie** [comment]
@META-DREAMER some exciting news, looks like this is already possible with a few workarounds 👀
https://twitter.com/pugson/status/1612487224058150913

I'll keep this thread updated if @pugson decides to reveal his secrets 🪄
